README.md

```diff
@@ -22,20 +22,31 @@ For a more detailed view of how we think about Lyra, check out our introductory
 
 ## Getting Started
 
+### Build
 The project requires [Go](https://golang.org/doc/install) 1.11 or higher, and [go modules](https://github.com/golang/go/wiki/Modules) to be on.
 
 1. Clone the git repo: `$ git clone https://github.com/lyraproj/lyra`
 2. Build the binary: `$ cd lyra; make lyra`
-3. Run the binary with the sample Workflow: ` $ ./build/lyra --debug apply sample`
 
-You can view the sample Workflow that is applied by the command above in `plugins/example.pp`.  This does not touch any real-world resources.
+### Deploying Workflows with CLI
 
-There is an AWS Workflow called `attach` in `plugins\attach.pp` (To run: ` $ ./build/lyra --debug apply attach`). This will use the default AWS region configured in `~/aws/config` and create real AWS resources.  To delete the Workflow (i.e. its resource), run ` $ ./build/lyra --debug delete attach`.  The `data.yaml` file can be used to configure AWS tag data.
+> **!! WARNING: THIS WORKFLOW CREATES REAL RESOURCES ($$) !!**
 
-## Integrating with Kubernetes
-1. Install the Lyra CRD as described in the [lyra-operator](https://github.com/lyraproj/lyra-operator/blob/master/README.md) repo
-2. Start Lyra in controller mode: ` $ ./build/lyra --debug controller`
-3. Create a Workflow resource, also described in the [lyra-operator](https://github.com/lyraproj/lyra-operator/blob/master/README.md) repo
+1. Run the binary with the [sample Workflow](https://github.com/lyraproj/lyra/blob/master/plugins/.yaml): ` $ ./build/lyra apply aws_vpc_yaml.yaml --debug`
+2. Delete the Workflow (i.e. its resources), run ` $ ./build/lyra delete aws_vpc_yaml --debug`.  
+
+This workflow is an AWS Workflow called `aws_vpc_yaml` in `plugins\aws_vpc_yaml.yaml`. This will use the default AWS credentials configured in your `~/.aws/credentials`. Region is currently hard-coded to `eu-west-1`. 
```

Comment:
I'm not sure the region bit of this is right.  We hard-code the region for terraform, but for the namespace Aws (rather than TerraformAws) I think we use the CLI's region (e.g. search for `eu-west-1` in the repo).  Could say something like:

This will use the default AWS credentials configured in your `~/.aws/credentials` and the default region configured in `~/.aws/config` (which can be checked using `aws configure get region` or set using `aws configure set region eu-central-1`)

Reply:
Oh that's right. Fixing.